### PR TITLE
MC-14388 Adds list site certificates to SP//

### PR DESCRIPTION
**Pull request by m-orfanos**
#### Changes made
 Adds docs for list site certificates

### CODE REVIEW

**Review by razine-bensari — APPROVED** [review]
looks good to me

File: source/includes/stackpath/_certificates.md [hunk 1]
```diff
@@ -1,5 +1,130 @@
...
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/certificates?siteId=<a href="#stackpath-sites">:siteId</a></code>
+
+Query Params | &nbsp;
+--- | ---
+`siteId`<br/>*UUID* | The ID of the site for which the certificate is applied to. This parameter is required.
+
+Attributes | &nbsp;
+--- | ---
+`id`<br/>*string* | The unique identifier of the certificate.
```
Comment (y0geshdev) [inline source/includes/stackpath/_certificates.md]: `id` type is string or UUID?

File: source/includes/stackpath/_certificates.md [hunk 2]
```diff
@@ -1,5 +1,130 @@
...
+      "id": "2a12237d-c388-4c19-bff1-b019b837e8f1",
+      "issuer": "",
+      "caBundle": "",
+      "status": "failed",
+      "expiresAt": "2021-04-01T18:11:11.903458Z",
+      "commonName": "helloworld.com",
+      "createdAt": "2021-04-01T18:11:11.904285Z",
+      "isManaged": true,
+      "trusted": false,
+      "updatedAt": "2021-04-04T18:30:32.021842Z"
```
Comment (y0geshdev) [inline source/includes/stackpath/_certificates.md:28]: If it's possible to have some realistic looking values for `issuer`, `caBundle` and `commonName` then that would be good.

File: source/includes/stackpath/_certificates.md [hunk 3]
```diff
@@ -1,5 +1,130 @@
...
+> The above command returns a JSON structured like this:
+```json
+{
+  "data": {
+    "id": "2a12237d-c388-4c19-bff1-b019b837e8f1",
+    "issuer": "",
+    "caBundle": "",
+    "status": "failed",
+    "expiresAt": "2021-04-01T18:11:11.903458Z",
+    "commonName": "helloworld.com",
```
Comment (y0geshdev) [inline source/includes/stackpath/_certificates.md:87]: same as above (if possible).

**Review by y0geshdev — APPROVED** [review]
Some nitpick but not blockers.

**Pull request merged**